apps/apps-cli/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@saleor/apps-cli",
+  "description": "",
+  "version": "0.0.0",
+  "author": "Saleor",
+  "scripts": {
+    "build": "pnpm bundle",
+    "generate": "graphql-codegen",
+    "lint": "prettier --write . && eslint src/**/*.ts --cache --fix",
+    "test": "pnpm vitest",
+    "typecov": "type-coverage --cache",
+    "typecov-report": "typescript-coverage-report",
+    "watch": "concurrently \"npm:watch-*\"",
+    "watch-esbuild": "esbuild --watch src/cli.ts --bundle --minify --outfile=dist/apps-cli.js --platform=node --format=esm --target=node18 --banner:js=\"import { createRequire } from 'module';const require = createRequire(import.meta.url);import { dirname } from 'path'; import { fileURLToPath } from 'url'; const __dirname = dirname(fileURLToPath(import.meta.url));\" --out-extension:.js=.js",
```

Comment:
Do we need this? I think its copy-pasted from saleor/cli and I don't want to replicate its patterns